cmd/auth.go

```diff
@@ -22,7 +22,7 @@ var authUpCmd = &cobra.Command{
 			fmt.Fprint(os.Stdout, err.Error())
 			return err
 		}
-		_, err = db.Exec("CREATE TABLE IF NOT EXISTS %s.%s (id serial, name text, username text unique, password text, metadata jsonb)", config.PrestConf.AuthSchema, config.PrestConf.AuthTable)
+		_, err = db.Exec(fmt.Sprintf("CREATE TABLE IF NOT EXISTS %s.%s (id serial, name text, username text unique, password text, metadata jsonb)", config.PrestConf.AuthSchema, config.PrestConf.AuthTable))
```

Comment:
`.Exec` receive params eg:
```go
db.Exec("CREATE TABLE IF NOT EXISTS ?.? (id serial, name text, username text unique, password text, metadata jsonb)", config.PrestConf.AuthSchema, config.PrestConf.AuthTable)
```

read more in https://go.dev/doc/database/change-data

Reply:
@avelino the change not made effect based on the instuction above, maybe a specified version is needed to run the change?

Reply:
Sorry is:
```go
db.Exec("CREATE TABLE IF NOT EXISTS $1.$2 (id serial, name text, username text unique, password text, metadata jsonb)", config.PrestConf.AuthSchema, config.PrestConf.AuthTable)
```